dockers/docker-database/base_image_files/sonic-db-cli

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
+import os
+import sys
+import swsssdk
+import logging
+import subprocess
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.NullHandler())
+
+DOCKER_EXEC_FLAGS = "i"
+
+# Determine whether stdout is on a terminal
+if os.isatty(1):
+    DOCKER_EXEC_FLAGS += "t"
+
+if len(sys.argv) < 3:
+    msg = "More than three arguments are required. 'usage: sonic-db-cli 'dbname' 'redis commands''"
```

Comment:
>'redis commands' [](start = 81, length = 16)

Try best to emulate `redis-cli`. It accepts arguments without quotation.

Reply:
it is the same as redis-cli usage now, there should be no quotation. update the usage string, remove the quotation in helper string.